### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Andyveloper**
# Take into account
- Please don't forget to seed the database as instructed in the README
## Changes and updates
- View will now be rendered properly
- When clicking on 'see all posts' or 'see all comments' it will display the right data
- Methods for most recent inputs are now being applied correctly

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+  <%else%>
+  <% @user.most_recent.each do |post| %>
+  <div>
+    <h3>Post #<%= post.id %></h3>
+    <p><%= post.text %></p>
+  </div>
+  <%end%>
+  <%end%>
+  <%= link_to 'See all posts', user_posts_path(@user.id) if @user.post_counter > 3%>
+</div>
```
Comment (ahmetbozaci) [inline app/views/users/show.html.erb]: - [ ] See all posts button is not appear in the page. Kindly fix it.

image

File: app/views/posts/_posts.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        </li>
+      <% end %>
+      </ul>
+      <%= link_to 'See all comments', user_post_path(@user.id, post.id) %>
+      <%end%>
+    </div>
+    <hr>
+    <%end%>
+  </div>
+</div>
```
Comment (ahmetbozaci) [inline app/views/posts/_posts.html.erb]: - [ ] Kindly add a pagination button on the posts page to follow wireframe.

image

File: db/seeds.rb [hunk 3]
```diff
@@ -1,9 +1,42 @@
...
+like= Like.create(author: second_user, post: post)
+like2= Like.create(author: third_user, post: post)
+like3= Like.create(author: third_user, post: post2)
+like4= Like.create(author: third_user, post: post3)
+like5= Like.create(author: third_user, post: post4)
+like6= Like.create(author: third_user, post: post5)
+like7= Like.create(author: second_user, post: post2)
+like8= Like.create(author: second_user, post: post3)
+like9= Like.create(author: first_user, post: post5)
+like10= Like.create(author: second_user, post: post5)
```
Comment (ahmetbozaci) [inline db/seeds.rb:42]: - [ ] Nice to add samples on the seed file. But it is not creating any sample. You can fix it. OPTIONAL
image

___
### EXAMPLE:
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Andyveloper  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
✔️ Your git-flow is correct. 
✔️ You set up linter files and no errors. 
✔️ Your pull request is very professional. It has a meaningful title and summary. Keep it up.
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by danielufeli — APPROVED** [review]
Hi @Andyveloper ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Design ✔️

## Optional suggestions
- [Optional] Kindly update your ReadMe 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**